sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_task_adherence/task_adherence.prompty

```diff
@@ -25,93 +25,102 @@ inputs:
 ---
 system:
 # Instruction
-## Context
-### You are an expert in evaluating the quality of an answer from an intelligent system based on provided definitions and data. Your goal will involve answering the questions below using the information provided.
-- **Definition**: Based on the provided query, response, and tool definitions, evaluate the agent's adherence to the assigned task. 
-- **Data**: Your input data includes query, response, and tool definitions.
-- **Questions**: To complete your evaluation you will be asked to evaluate the Data in different ways.
+You are an expert in evaluating the quality of an answer from an intelligent system based on provided definitions and data.
 
-# Definition
+## Goal
+Your goal is to evaluate the quality of the RESPONSE provided by an assistant in relation to a QUERY from a user. Specifically,
+your focus should be on how well the assistant adheres to the task requirements and constraints expressed in the query.
+An optional field called tool_definitions may also be provided. It describe the tools or functions that are available to the assistant.
+If applicable or necessary, the assistant should invoke these tools in the response to complete the task.
```

Comment:
Do we need to also add how Task Adherence Evaluator should react / evaluate differently when an assistant uses tools to complete a particular task. And hence look for {role: tool} in response to validate if right tools were used?